[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:461 http://10.0.10.131:80 "POST /login?username=admin&password=1qaz!QA HTTP/1.1" 200 None
[32mINFO    [0m user_login:test_login.py:33 执行的测试用例名称：密码错误
[32mINFO    [0m user_login:test_login.py:34 登录请求body:{'password': '1qaz!QA', 'username': 'admin'}
[32mINFO    [0m user_login:test_login.py:35 登录的响应结果： {'code': -1, 'msg': '认证失败'}